### PR TITLE
Prompt display order resolved

### PR DESCRIPTION
**Pull request by GWill44**
Fixed.
[https://github.com/GWill44/login-page/issues/6](url)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -57,26 +56,28 @@ function isRegisterFormValid() {
   //Form field null validation
   let formValid = true;
 
-  //"Make the first input error be displayed byt the registerStatus"
-  registerFields.forEach((item, i) => {
-    console.log(item);
+  //Verifies fields arent empty
+  registerFields.every((item, i) => {
```
Comment (coeymusa) [inline main.js:60]: formValid = registerFields.every.....

File: main.js [hunk 2]
```diff
@@ -57,26 +56,28 @@ function isRegisterFormValid() {
...
     if(document.getElementById(item.name).value === "") {
       document.getElementById('registerStatus').innerHTML = `${item.label} field cannot be empty`;
       formValid = false;
-    }
+      return false;
```
Comment (coeymusa) [inline main.js:64]: not needed

File: main.js (hunk 1 above)
Comment (coeymusa) [reply main.js:60]: const registerFields = [
    {name:"registerUsername",label:"Username"},
    {name:"registerFirstName",label:"First name"},
    {name:"registerLastName",label:"Last name"},
    
const found = registerFields.find(element => document.getElementById(item.name).value == "");

File: main.js [hunk 3]
```diff
@@ -57,26 +56,28 @@ function isRegisterFormValid() {
...
-  const password1 = document.getElementById('registerPassword').value;
-  const password2 = document.getElementById('registerRetypePassword').value;
-  if(password1 === password2){
-    document.getElementById('registerStatus').innerHTML = "Passwords Match";
-    return true;
-  }else{
-    document.getElementById('registerStatus').innerHTML = "Passwords do not match";
-    return false;
+  //Returns a false value if the field verification returns false
+  if (formValid === false){ return false}
```
Comment (coeymusa) [inline main.js:69]: document.getElementById('registerStatus').innerHTML = `${found.name} field cannot be empty`;

File: main.js [hunk 4]
```diff
@@ -57,26 +56,28 @@ function isRegisterFormValid() {
...
+  //Password comparison
+  else if (true) {
```
Comment (coeymusa) [inline main.js:71]: remove

**Comment by GWill44** [comment]
I've pushed the last commit with your suggested changes.

**Pull request merged**